plugins/obs-ffmpeg/CMakeLists.txt

```diff
@@ -14,7 +15,11 @@ find_package(
 
 add_library(obs-ffmpeg MODULE)
 add_library(OBS::ffmpeg ALIAS obs-ffmpeg)
-add_subdirectory(ffmpeg-mux)
```

Comment:
Is this normal ? It prevents CMake to generate a `obs-ffmpeg-mux` binary.

Reply:
goof